man/logind.conf.xml

```diff
@@ -212,6 +212,14 @@
         <xi:include href="version-info.xml" xpointer="v256"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>HandleSecureAttentionKey=</varname></term>
+
+        <listitem><para>Specifies if the Secure Attention Sequence (Ctrl+Alt+Shift+Esc) should be handled. Defaults to <literal>secure-attention-key</literal>.</para>
```

Comment:
not a sequence, as above. it's a key combination

Reply:
First you say *if*, which suggests this was a boolean. But it's not, it's an enum. 

Better fold this into the next section.

Reply:
OK, I thought I had do to this because I had to specify the new version, however, seeing that `HandleLidSwitchExternalPower` was introduced, and just added there, I will do the same